classes/module/Module.php

```diff
@@ -1346,6 +1346,23 @@ public static function getInstanceByName($module_name)
         return static::$_INSTANCE[$module_name];
     }
 
+    public static function getInvalidityErrors(string $moduleName): array
+    {
+        $validityErrors = [];
+
+        if (!Validate::isModuleName($moduleName)) {
+            $validityErrors[] = $moduleName . ' module name is invalid';
+        }
```

Comment:
Hi @jolelievre ,
If module name is not valid, I think we can return imediatly the validityErrors to improve the perfs.

also the `Module::getInstanceByName` already includes this check is triggered an exception when we are in development mode